llm/sglang/sglang.yaml

```
@@ -0,0 +1,34 @@
+service:
+  # Specifying the path to the endpoint to check the readiness of the service.
+  readiness_probe: /health
+  # How many replicas to manage.
+  replicas: 2
+
+envs:
+  MODEL_NAME: meta-llama/Llama-2-7b-chat-hf
+  HF_TOKEN: <your-huggingface-token> # Change to your own huggingface token
+
+resources:
+  accelerators: {L4:1, A10G:1, A10:1, A100:1, A100-80GB:1}
+  ports:
+    - 8000
+
+setup: |
+  conda activate sglang
+  if [ $? -ne 0 ]; then
+    conda create -n sglang python=3.10 -y
+    conda activate sglang
+  fi
+
+  pip list | grep sglang || pip install "sglang[all]"
+  pip install git+https://github.com/huggingface/transformers
```

Let's pin the version of `transformers` to make this example more stable (We did this change in #3118 for vLLM as well).
```suggestion
  pip list | grep transformers || pip install transformers==4.37.2
```